src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,240 @@
+            </>,
+          );
+          close();
+
+          eventQueue.set(
+            operation.metadata.id,
+            async () => {
+              //Start Instance if instance created successfully.
+              const instanceToStart = fetchInstance(
+                values.instanceName,
+                values.project,
+              );
+
+              //Display starting notification
+              toastNotify.info(
+                <>
+                  Created instance <b>{values.instanceName}</b>, now starting
+                  it.
+                </>,
+              );
+
+              startInstance(await instanceToStart)
+                .then(() => {
+                  //If instance started successfully, display success message as shown in design.
+                  notifySuccess(values.instanceName);
+                })
+                .catch((e) => {
+                  toastNotify.failure(
+                    `Unable to start instance "${values.instanceName}`,
+                    e,
+                  );
+                });
+            },
+            (msg) => {
+              toastNotify.failure(
+                `Instance creation for "${values.instanceName}" failed.`,
+                new Error(msg),
+              );
+            },
+          );
+        })
+        .catch((e) => {
+          toastNotify.failure(
+            `Instance creation for "${values.instanceName}" failed.`,
+            e,
+          );
+          close();
+        });
+    },
+  });
+
+  return (
+    <Modal
+      close={close}
+      title="Duplicate Instance"
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={close}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={formik.isSubmitting}
+            disabled={!formik.isValid}
+            onClick={() => void formik.submitForm()}
+          >
+            Duplicate & start
+          </ActionButton>
+        </>
+      }
+    >
+      <Form onSubmit={formik.handleSubmit}>
+        <Input
+          {...formik.getFieldProps("instanceName")}
+          type="text"
+          label="New instance name"
+        />
+        {isClustered ?? (
+          <Select
+            {...formik.getFieldProps("targetClusterMember")}
+            id="targetClusterMember"
+            label="Target cluster member"
+            options={clusterMembers.map((clusterMember) => {
+              return {
+                label: clusterMember.server_name,
+                value: clusterMember.server_name,
+              };
+            })}
+          />
+        )}
+        <Select
+          {...formik.getFieldProps("storagePool")}
+          id="storagePool"
+          label="Storage pool"
```

Comment:
let's call this "Target storage pool", also please adjust the variable names please